runtime/runtime_test.go

```diff
@@ -302,6 +303,38 @@ func TestCheckOPAUpdateLoopWithNewUpdate(t *testing.T) {
 	testCheckOPAUpdateLoop(t, baseURL, "OPA is out of date.")
 }
 
+func TestCheckAuthIneffective(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	var stdout bytes.Buffer
```

Comment:
A bit of a nit, but OPA logs to stderr, so this should either be renamed `stderr` or I guess simply just `output` as we're really just printing to a buffer for the test.